ftw/sliderblock/profiles/default_plone5/registry.xml

```diff
@@ -25,21 +25,12 @@
         </value>
     </record>
 
-    <records prefix="plone.resources/ftw-sliderblock-js"
-             interface='Products.CMFPlone.interfaces.IResourceRegistry'>
-        <value key="js">++resource++ftw.sliderblock/sliderblock.js</value>
-        <value key="deps">jquery</value>
-    </records>
-
     <records prefix="plone.bundles/ftw-sliderblock-resources"
              interface='Products.CMFPlone.interfaces.IBundleRegistry'>
-        <value key="resources">
-            <element>ftw-sliderblock-js</element>
-        </value>
         <value key="enabled">True</value>
         <value key="depends">plone-legacy</value>
         <value key="compile">False</value>
-        <value key="jscompilation">++plone++sliderblock/ftw-sliderblock-compiled.js</value>
+        <value key="jscompilation">++resource++ftw.sliderblock/sliderblock-compiled.js</value>
         <value key="merge_with">default</value>
```

Comment:
According to https://devdocs.4teamwork.ch/plone/javascript/ftw-resource-bundling-plone5/#declare-a-bundle-that-is-built-externally you need a `last_compilation` entry.

Reply:
Done